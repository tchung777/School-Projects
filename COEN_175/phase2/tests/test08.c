int main(void)
{
    int i, j, k;
    int *p, **q;

    i = j + 1;

    p = &i;
    q = &p;
    i = - - - - - - - - - - k;
    j = !!!!!!!!!!!!!!k;
}
