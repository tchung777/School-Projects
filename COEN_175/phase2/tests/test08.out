add
addr
addr
neg
neg
neg
neg
neg
neg
neg
neg
neg
neg
not
not
not
not
not
not
not
not
not
not
not
not
not
not
